examples/example_notebooks/GEM_cookbook.ipynb

```diff
@@ -0,0 +1,1541 @@
+     "text": [
+      "/home/wilhelmk/tools/anaconda3/envs/tf-cpu/lib/python3.7/site-packages/gym/logger.py:30: UserWarning: \u001b[33mWARN: Box bound precision lowered by casting to float32\u001b[0m\n",
+      "  warnings.warn(colorize('%s: %s'%('WARN', msg % args), 'yellow'))\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "<gym_electric_motor.envs.gym_pmsm.perm_mag_syn_motor_env.DiscPermanentMagnetSynchronousMotorEnvironment at 0x7fda31b714d0>"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import gym_electric_motor as gem\n",
+    "\n",
+    "basic_env = gem.make(\"PMSMDisc-v1\")  # pass a motor environment ID \n",
+    "basic_env"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 2.2  Physical System\n",
+    "Each system consists of a voltage supply, a power electronic converter, an electrical motor, and the mechanical load (SCML) as shown in the above figure. Each such SCML-system is simulated by a user-defined ODE solver.\n",
+    "\n",
+    "#### Voltage Supply\n",
+    "\n",
+    "The voltage supply module provides both, DC and AC voltage supplies. \n",
+    "- The DC supplies are either ideal or non-ideal voltage sources.\n",
+    "- The AC supplies are either ideal single phase or ideal three-phase AC sources.\n",
+    "\n",
+    "More documentation regarding the voltage supplies of GEM can be found [here](https://upb-lea.github.io/gym-electric-motor/parts/physical_systems/voltage_supplies/voltage_supply.html). \n",
+    "For the PMSM environment example, a non-ideal DC voltage supply is created. \n",
+    "Here, the DC-link is modeled as an RC-circuit loaded from an ideal DC voltage source. \n",
+    "\n",
+    "The non-ideal DC supply in GEM is named 'RCVoltageSupply' and the supply_parameter(dict) consists of resistance R in ohm and capacitance C in farad\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "supply = 'RCVoltageSupply'\n",
+    "supply_parameter=dict(R=10, C=4e-3)  # Note, R and C values here are not realistic."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Converter \n",
+    "The converters are divided into two classes: The continuously controlled and the discretely controlled converters.\n",
+    "\n",
+    "In the continuous case, the agent's actions denote a duty cycle that is modulated to the converter's output voltage.\n",
+    "\n",
+    "In the discrete case, the agent's actions denote switching states of the converter at the given instant.\n",
+    "Here, only a discrete amount of options are available. In this notebook, for the PMSM the discrete B6 bridge converter with six switches is utilized per default. This converter provides a total of eight possible actions.\n",
+    "![](../../docs/plots/B6.svg)\n",
```

Comment:
Unfavorable line break, because the image seems to be integrated as floating environment. In my notebook the last part of this sentence was only displayed behind the picture, i.e. the picture divides the sentence.